src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Jobs/StopAzureRmRecoveryServicesBackupJob.cs

```diff
@@ -24,18 +24,27 @@
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
 {
+    /// <summary>
+    /// Cancels a job. Returns the corresponding job object after this operation finishes.
```

Comment:
*after trigger of cancellation finishes. It need not cancel it successfully. It will only notify the service that a cancellation has been triggered. Please convey this in any way you like. :)